mods/corporate/datum/robolimbs.dm

```diff
@@ -78,9 +53,54 @@
 	icon = 'icons/mob/human_races/cyberlimbs/veymed/veymed_main.dmi'
 	can_eat = 1
 	skintone = 1
-	unavailable_at_fab = 1
 
 /datum/robolimb/shellguard
 	company = "Shellguard"
 	desc = "This limb has a sturdy and heavy build to it."
 	icon = 'icons/mob/human_races/cyberlimbs/shellguard/shellguard_main.dmi'
+
+/datum/robolimb/economy
+	company = "Ward-Takahashi Econ."
+	desc = "A simple robotic limb with retro design. Seems rather stiff."
+	icon = 'icons/mob/human_races/cyberlimbs/wardtakahashi/wardtakahashi_economy.dmi'
+
+/datum/robolimb/xion
+	company = "Xion"
+	desc = "This limb has a minimalist black and red casing."
+	icon = 'icons/mob/human_races/cyberlimbs/xion/xion_main.dmi'
+
+/datum/robolimb/nanotrasen
+	company = "NanoTrasen"
+	desc = "This limb is made from a cheap polymer."
+	icon = 'icons/mob/human_races/cyberlimbs/nanotrasen/nanotrasen_main.dmi'
+
+#define DEFINE_ROBOLIMB_DESIGNS(MODEL_PATH, MODEL_ID, MODEL_NAME)         \
```

Comment:
I think this macro should go into the main robolimb file, not mod's one. Not really corp specific is it.